index.js

```diff
@@ -1,29 +1,53 @@
 'use strict';
+var regexps = require('./regexps.json');
+const dayMilliSeconds = 24 * 60 * 60 * 1000;
```

Comment:
Isn't the `const` keyword not available before node 4.0? I would rather we use `var` here. That will help us maintain support for older versions as well.